src/legacy/ui/public/vis/editors/default/agg.js

```diff
@@ -51,6 +52,35 @@ uiModules
           $scope.aggIsTooLow = calcAggIsTooLow();
         });
 
+        if ($scope.agg.schema.group === 'buckets') {
+          $scope.$watchMulti([
+            '$last',
+            'lastParentPipelineAggTitle',
+            'agg.type'
+          ], function ([isLastBucket, lastParentPipelineAggTitle, aggType]) {
+            $scope.error = null;
+            $scope.disabledParams = [];
+
+            if (!lastParentPipelineAggTitle || !isLastBucket || !aggType) {
+              return;
+            }
+
+            if (['date_histogram', 'histogram'].includes(aggType.name)) {
+              $scope.onAggParamsChange(
+                $scope.agg.params,
+                'min_doc_count',
+                aggType.name === 'histogram' ? true : 0);
```

Comment:
Please leave a comment what the difference is between `true` and `0` values and why we pass boolean for `histogram` and number for `date_histogram`